drake/lcmtypes/lcmt_lidar_point.lcm

```diff
@@ -0,0 +1,28 @@
+package drake;
+
+struct lcmt_lidar_point
+{
+  /// Location of the point in the sensor's coordinate system.
+  /// \{
+  float x;
+  float y;
+  float z;
+  /// \}
+
+  /// Strength of the return (in device dependent units).
+  float intensity;
```

Comment:
This would be relevant for a velodyne but not for a planar lidar or a depth camera,

consider having a arrays of attributes rather than array of points with hard coded attributes.